src/stdlib/Json.ts

```diff
@@ -0,0 +1,127 @@
+import { AssociativeArray } from "../brsTypes/components/AssociativeArray";
+import { BrsArray } from "../brsTypes/components/BrsArray";
+import { Interpreter } from "../interpreter";
+import { Literal } from "../parser/Expression";
+import { randomBytes } from "crypto";
+
+import {
+    BrsBoolean,
+    BrsInvalid,
+    BrsString,
+    BrsType,
+    Callable,
+    Float,
+    Int32,
+    Int64,
+    ValueKind
+} from "../brsTypes";
+
+function brsValueOf(x: any): any {
```

Comment:
This function returns a `BrsType`, not `any` – being strict with what functions return will help to prevent weird type-checking errors!  Looks like there's one change required to make that compile, which I'll comment on below.

Reply:
done